ServiceDocumentSamples/README.md

```diff
@@ -1,6 +1,6 @@
-# Service Document Sample Components
+# Service Document Samples
 
-This project contains sample components can be used on the service document template. Through Salesforce DX project, these sample components can be pushed to your org and used on service document templates.
+This project contains samples including components and triggers can be used on the service document template. Through Salesforce DX project, these sample components and triggers can be pushed to your org and used on service document templates.
```

Comment:
I feel we can rephrase the wording here. 

> This project contains sample codes for LWC components and Apex triggers which can be used on the service document template. 

What do you think?

Reply:
Update.